pkg/reader/ccip.go

```diff
@@ -775,6 +775,81 @@ func (r *ccipChainReader) GetContractAddress(contractName string, chain cciptype
 	return addressBytes, nil
 }
 
+// GetLinkPriceUSD gets the LINK price in USD from the FeeQuoter contract on the destination chain.
+func (r *ccipChainReader) GetLinkPriceUSD(ctx context.Context) (cciptypes.BigInt, error) {
+	// Exit without an error if we cannot read the destination.
+	if err := validateExtendedReaderExistence(r.contractReaders, r.destChain); err != nil {
+		return cciptypes.BigInt{}, fmt.Errorf("validate extended reader existence: %w", err)
+	}
+
+	feeQuoterCfg, err := r.getDestFeeQuoterStaticConfig(ctx)
+	if err != nil {
+		return cciptypes.BigInt{}, fmt.Errorf("get destination fee quoter static config: %w", err)
+	}
+
+	linkTokenAddr := feeQuoterCfg.LinkToken
+
+	getLinkPriceUSD, err := r.getLinkPriceUSD(ctx, linkTokenAddr)
+
+	if err != nil {
```

Comment:
Inline and reduce newlines

```suggestion
	getLinkPriceUSD, err := r.getLinkPriceUSD(ctx, feeQuoterCfg.LinkToken)
	if err != nil {
```